packages/core-dialog/core-dialog.jsx

```diff
@@ -1,5 +1,25 @@
 import React from 'react'
+import dialog from './core-dialog'
+import {IS_BROWSER, exclude} from '../utils'
 
-module.exports = () => (
-  <div>Testing dialog</div>
-)
+const NO_SUPPORT = IS_BROWSER && typeof window.HTMLDialogElement === 'undefined'
+const DEFAULTS = {hidden: null, onToggle: null}
+if (NO_SUPPORT) DEFAULTS.open = null
```

Comment:
Why remove open when `NO_SUPPORT`? this will be wrong when server-side-rendering :)

Reply:
My bad. It was something I did when `showModal()` complained about open attribute already set. But fixed it in core-dialog.js instead. I will remove this part in core-dialog.jsx